wayshot/src/config.rs

```diff
@@ -0,0 +1,96 @@
+use crate::utils::EncodingFormat;
+use serde::{Deserialize, Serialize};
+use std::{env, fs::File, io::Read, path::PathBuf};
+use tracing::Level;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Config {
+    pub screenshot: Option<Screenshot>,
+    pub fs: Option<Fs>,
```

Comment:
Again not a fan of the fs terminology as it seems vague but I think we can get past this.

Reply:
I think we can name it `file` as with `bool` switch, if it's OK to you